src/css/custom.scss

```diff
@@ -256,3 +256,25 @@
 .navbar__logo {
   width: 2rem;
 }
+
+// TODO: Test these on mobile
+details.api-history {
+  transform: translateY(
+    calc(var(--ifm-heading-vertical-rhythm-bottom) * var(--ifm-leading) * -0.5)
+  );
+
+  summary {
+    width: fit-content;
+    font-weight: normal;
+    font-style: italic;
+
+    &:hover {
+      cursor: pointer;
+    }
+  }
+
+  table {
+    margin-top: 0.5rem;
+    margin-bottom: 0;
+  }
+}
```

Comment:
As discussed on our call, let's move these to `src/components/ApiHistoryTable.module.scss` to match the pattern we're using with `src/components/APIStructurePreview.tsx`.

Reply:
Resolved in [`d9e15cb` (#594)](https://github.com/electron/website/pull/594/commits/d9e15cb575ea94a5061566127e9bec1c2d3958ca)